test/Microsoft.TestPlatform.AcceptanceTests/EventLogCollectorTests.cs

```diff
@@ -63,9 +61,9 @@ public void EventLogDataCollectorShoudCreateLogFileWithoutEventsIfEventsAreNotLo
         StdErrorDoesNotContains("Unable to read event log");
     }
 
-    private string GetRunsettingsFilePath(TempDirectory tempDirectory)
+    private string GetRunsettingsFilePath(TempDirectory TempDirectory)
```

Comment:
I don't think you want this rename. Also, do we want to give the possibility to pass a temp dir or shall we use the base property directly within the method?